bin/build_utils_gitlab

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -e
+
+[[ "$TRACE" ]] && set -x
+
+CURRENT_DIR=$(cd $(dirname ${BASH_SOURCE[0]}); pwd)
+
+# source build utilities
+pushd ${CURRENT_DIR}
+  . ./build_utils
+popd || true
+
+CONTAINER_IMAGES="
+  secretless-broker
+  secretless-dev
+  secretless-broker-quickstart
+"
```

Comment:
You should probably use an array here instead of a long string

Reply:
addressed